Content.Server/Body/Components/StomachComponent.cs

```diff
@@ -47,6 +47,12 @@ public sealed partial class StomachComponent : Component
         [DataField]
         public EntityWhitelist? SpecialDigestible = null;
 
+        /// <summary>
+        ///     A bool for if the diet is exclusive to a whitelist. If they have additional foods they can eat on top of the human diet, that would be false. Else, it is true.
```

Comment:
```suggestion
        /// Controls whitelist behavior. If true, this stomach can digest <i>only</i> food that passes the whitelist. If false, it can digest normal food <i>and</i> any food that passes the whitelist.
```
Reworded for clarity.